### PR TITLE
#636: LengthOfTest.java:89-91: Fix calculation of length for

### PR DESCRIPTION
**Pull request by proshin-roman**
Pull request for the issue #636

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#701](https://github.com/yegor256/cactoos/issues/701) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#701](https://github.com/yegor256/cactoos/issues/701) is assigned to @neonailol[/z](https://www.0crat.com/u/neonailol), here is [why](https://www.0crat.com/footprint/C63314D6Z/520720353). The budget is 15 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§27](http://datum.zerocracy.com/pages/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/main/java/org/cactoos/io/LengthOf.java [hunk 1]
```diff
@@ -65,6 +65,11 @@ public LengthOf(final Input input) {
     )
     public LengthOf(final Input input, final int max) {
         super(() -> {
+            if (max == 0) {
+                throw new IllegalArgumentException(
+                    "Cant use a buffer limited to zero size"
```
Comment (svendiedrichsen) [inline src/main/java/org/cactoos/io/LengthOf.java]: Typo: either `can't` or `cannot` please

Comment (proshin-roman) [reply src/main/java/org/cactoos/io/LengthOf.java]: @svendiedrichsen thanks, I've fixed it

**Comment by proshin-roman** [comment]
@neonailol could you take a look on this PR, please? I have a day only to close the ticket.

**Comment by proshin-roman** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting ([here](https://github.com/yegor256/cactoos/issues/701#issuecomment-368219795))

@proshin-roman It's a code review job [#701](https://github.com/yegor256/cactoos/issues/701), can't put it on hold

**Comment by neonailol** [comment]
@yegor256 this can be merged

**Comment by proshin-roman** [comment]
@yegor256 could you merge (or decline) this pull request, please? It shouldn't take too long to review.

**Comment by 0crat** [comment]
@neonailol[/z](https://www.0crat.com/u/neonailol) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job, as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by ypshenychka** [comment]
@proshin-roman According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>Pull request description explains the solution proposed and contains a link to the original ticket it is related to.

Please add more details into description.

**Comment by ypshenychka** [comment]
@neonailol According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one cosmetic issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ypshenychka** [comment]
@svendiedrichsen According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>Messages in a job [always](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html#4-avoid-noiseaddress-your-comments) start with a name of a user they are addressed to.

Please correct your code review comment by indicating an addressee in the beginning.

**Comment by proshin-roman** [comment]
@ypshenychka I've changed the description of the pull request. Just btw about found problems: the change is too small to find more problems :)

**Comment by ypshenychka** [comment]
@proshin-roman Thank you. I understand, but currently we have what we have. I mean the rules :) Probably in future small PRs won't have to pass QA review.

**Comment by ypshenychka** [comment]
@neonailol ping

**Comment by ypshenychka** [comment]
@neonailol Waiting on your [confirmation](https://github.com/yegor256/cactoos/pull/701#issuecomment-368551243). Otherwise you can lose your payment when 10 days of assignment are behind.

**Comment by neonailol** [comment]
@ypshenychka i confirm

**Comment by ypshenychka** [comment]
@neonailol thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
`QA` review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka), total is [+877](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @neonailol[/z](https://www.0crat.com/u/neonailol), total is [+359](https://www.0crat.com/u/neonailol)

**Comment by 0crat** [comment]
> @0crat quality acceptable ([here](https://github.com/yegor256/cactoos/issues/701#issuecomment-369240919))

@ypshenychka The job [#701](https://github.com/yegor256/cactoos/issues/701) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+11044](https://www.0crat.com/u/yegor256)

**Pull request merged**